### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by ekoeryanto**
use pretest npm hook instead off `&&`

### CODE REVIEW

**Comment by cemremengu** [comment]
Your fork looks outdated, can you rebase your changes?

**Comment by ekoeryanto** [comment]
@cemremengu, I don't see it outdated. which branch should I rebase?

**Comment by cemremengu** [comment]
@ekoeryanto sorry my bad. That part of README is actually out of sync with `package.json`.

Can you please update the package.json of the project to use `pretest` as well and sync the test scripts with README?

File: README.md [hunk 1]
```diff
@@ -189,8 +189,8 @@ It is a standard Fastify plugin and you will not need to add the `listen` method
 }
 
 "scripts": {
-- "test": "tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
-+ "test": "standard && tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
++ "pretest": "standard",
+  "test": "tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
   "start": "fastify start -l info app.js",
   "dev": "fastify start -l info -P app.js",
```
Comment (ekoeryanto) [inline README.md]: I think the standard `dev` script is to use --watch

Comment (cemremengu) [reply README.md]: I am not sure if everyone will prefer that. @mcollina wdyt?

Comment (mcollina) [reply README.md]: I'm very -1 on `dev` using watch.

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "lint": "standard",
     "unit": "tap test/*.test.js app_template/test/*/*.test.js",
-    "test": "npm run lint && npm run unit"
+    "pretest": "npm run lint",
+    "test": "npm run unit"
```
Comment (mcollina) [inline package.json]: I think you can just rename `unit` to `test` now.

**Pull request merged**